src/main/java/se/sundsvall/supportmanagement/service/scheduler/emailreader/EmailProcessingHealthIndicator.java

```diff
@@ -0,0 +1,41 @@
+package se.sundsvall.supportmanagement.service.scheduler.emailreader;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Component
+public class EmailProcessingHealthIndicator implements HealthIndicator {
+
+	private final AtomicBoolean healthy = new AtomicBoolean(true);
+	private final AtomicBoolean errors = new AtomicBoolean(false);
+
+	@Override
+	public Health health() {
+		if(healthy.get()) {
+			return Health.up().build();
+		} else {
```

Comment:
redundant else. This row will be sufficient (without the else)
`return Health.status("RESTRICTED").build();`